assets/styles/mentor-register-form.scss

```diff
@@ -0,0 +1,289 @@
+.step-1-mentor , .step-2-mentor , .step-3-mentor , .step-4-mentor , .step-5-mentor {
```

Comment:
maybe it's not necessary to rewrite all the style that's already been there with student form ?